Getting error `cannot find module` build/src/server

**Describe the bug**
Getting error `cannot find module` build/src/server

**Additional details**

module.js:550
    throw err;
    ^

Error: Cannot find module '../build/src/server'
    at Function.Module._resolveFilename (module.js:548:15)
    at Function.Module._load (module.js:475:25)
    at Module.require (module.js:597:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/Users/shanejonas/openrpc/generator-mock-server/bin/cli.js:3:16)
    at Module._compile (module.js:653:30)
    at Object.Module._extensions..js (module.js:664:10)
    at Module.load (module.js:566:32)
    at tryModuleLoad (module.js:506:12)
    at Function.Module._load (module.js:498:3)

--- a/bin/cli.js
+++ b/bin/cli.js
@@ -1,6 +1,6 @@
 #!/usr/bin/env node
 const program = require('commander');
-const server = require('../build/src/server');
+const server = require('../build/server');
 
 program
   .version(require('./get-version'))

